packages.json

```diff
@@ -77,7 +78,6 @@
 				"nautilus-open-any-terminal",
 				"openssh-askpass",
 				"simple-scan",
-				"yaru-theme",
```

Comment:
Can we keep it on disk? I'd still be useful for those who prefer to keep bluefin how it used to be.

Reply:
Yeah, sure.  Will add it back.
FWIW, if you like the Files icon, we can use only Yaru's icons